### PR TITLE
Issue112: Basic student analytics

### LINKED ISSUE #112: Add first level of analytics using single card values to the student

### PR DESCRIPTION
**Pull request by esteeroramy**
Fixes #112 
image

### CODE REVIEW

File: public/js/analytics.js [hunk 1]
```diff
@@ -81,6 +81,9 @@ var displayClassStatistics = function() {
     // Card visibilty
     $('#student-analytics-card').addClass('hidden');
     $('#class-analytics-card').removeClass('hidden');
+
+    $('#studentAnalyticsHeadr').addClass('hidden');
```
Comment (esteeroramy) [inline public/js/analytics.js]: `headr` => `header` lol, will do that for all instances

File: public/js/analytics.js [hunk 2]
```diff
@@ -93,236 +96,181 @@ var displayStudentStatistics = function(studentId) {
...
+
     var path = studentId ? '/studentAnalytics?studentId=' + studentId : '/studentAnalytics';
 
     // Request statistics
-    getQuestionsAnsweredVsClass(path);
-    getAccuracyVsClass(path);
-    getPointsVsClass(path);
-    getRatingVsClass(path);
+
+    // Student Ratings
```
Comment (esteeroramy) [inline public/js/analytics.js]: `ratings` => `statistics`

**Review by ahmedqarmout2 — APPROVED** [review]

**Review by Mohamed-A-Mohamed — APPROVED** [review]
Looks good, nice work.

**Review by Mohammed-JS-Khan — APPROVED** [review]
Quais

**Pull request merged**